examples/regression/e2e/tests/fixtures/find.rs

```diff
@@ -19,3 +17,8 @@ pub async fn link_with_text(client: &Client, text: &str) -> Result<Element> {
         .expect(format!("Link not found by `{}`", text).as_str());
     Ok(link)
 }
+
```

Comment:
While reading this .. I noticed a little simplification ... its in this PR 

https://github.com/leptos-rs/leptos/pull/4121